Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+serde = {version = "1.0.102", features = ["derive"]}
```

Comment:
Ideally this would optionally depend on the `dclabel` feature or, another, `serde` feature since this requires the `alloc` crate, which may or may not be available. But optional dependencies based on feature flags is only in stable cargo as of yesterday, so let's delay for a bit.